.travis.yml

```diff
@@ -0,0 +1,82 @@
+---
+
+###
+### Travis settings
+###
+sudo: required
+services:
+  - docker
+
+
+###
+### Env variables
+###
+env:
+  global:
+    - image=flaconi/java-base
+
+jobs:
+  include:
+    ###
+    ### Stage 1: Static file analysis
+    ###
+    - stage: static file analysis
+      install:
+        - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get update; then break; else i=$((i+1)); fi done
+        - max=100; i=0; while [ $i -lt $max ]; do if sudo apt-get -y install moreutils; then break; else i=$((i+1)); fi done
+        - git clone https://github.com/cytopia/awesome-ci /tmp/awesome-ci
+        - sh -c "cd /tmp/awesome-ci && ./configure --prefix=/usr/local && sudo make install"
+      script:
+        - ./tests/static-file-checks.sh
+
+    ###
+    ### Stage 2: Build, Test and Deploy
+    ###
+
+    # [Stage 2] [Job 1]
+    - stage: build and deploy
+      env: java=stretch-oracle-java8
```

Comment:
is there a reason for not using the build matrix here? I see no difference between the commands used for 8 and 9. we can DRY the code and we can speed up the job with parallelism if we use the [Travis build matrix](https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix).

Reply:
Yes, build matrices don't expand on multi-stages (yet). If you find a way to way build stages with matrix expansion I will invite you to a beer :beer:

Reply:
I see the legitimacy in having separate stages, but do we actually need them here?
I think having a build matrix here would have more benefits than what the stages (currently) offer us. what do you think?

Reply:
The need is purely to serve as a reference for how to actually do it. I agree matrix expansion is so much nicer, but let's have at least one repository covering build stages to copy from.

Reply:
I see. wasn’t aware of that reasoning :)